Menu/Menu.js

```diff
@@ -1,10 +1,20 @@
 
 const toggleMenu = () => {
   // Toggle the "menu--open" class on your menu refence. 
+  menu.classList.toggle("menu--open");
+  menu.classList.toggle("slide-right");
```

Comment:
looks like the animation works differently then you intended it to work. it is sliding from the right to the left.